crates/ironrdp-web/src/session.rs

```diff
@@ -353,6 +344,16 @@ pub(crate) enum RdpInputEvent {
     FastPath(FastPathInputEvents),
 }
 
+enum CursorStyle {
+    Default,
+    Hidden,
+    Url {
+        data: String,
+        hotspot_x: u16,
+        hotspot_y: u16,
+    },
+}
```

Comment:
Suggestion: it’s also possible to export an enum for TypeScript / JavaScript (see IronRdpErrorKind). It would be useful to have a stronger type there too. It’s not possible to use algebraic data types like in Rust, but at least we get something similar to `match` for the kind.